Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/child_app/view/MainActivity.java

```diff
@@ -36,4 +36,16 @@ private void setUpView() {
         setContentView(R.layout.child_app_activity_main);

```

Comment:
shouldn't it be changed to finish screen layout ?

Reply:
That's not what this line does. This one could be used for the initial screen, containing, for instance, a logo or such. Finish screen is set below that, in L45.